device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/auth/IotHubSasTokenHardwareAuthenticationProvider.java

```diff
@@ -83,40 +81,6 @@ public boolean canRefreshToken()
         return true;
     }
 
-    /**
-     * Getter for SSLContext
-     * @throws IOException if an error occurs when generating the SSLContext
-     * @return The value of SSLContext
-     */
-    @Override
-    public SSLContext getSSLContext() throws IOException
```

Comment:
Can just use the implementation defined in the super class, so no need to override here